test/test_encounters.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime, timedelta, timezone
+
+import pytest
+
+from welkin.exceptions import WelkinHTTPError
+from welkin.models.encounter import Encounter, Encounters
+
+UTC = timezone.utc
+
+
+@pytest.mark.vcr()
+def test_encounter_create(client, vcr_cassette):
+    patient = client.Patient(id="371dd15c-cedc-4425-a394-d666c8d3fc01")
+    start = datetime.now(tz=UTC) + timedelta(hours=6)
+    end = start + timedelta(hours=1)
+
+    data = {
+        "title": "new enc",
+        "description": "new encounter",
+        "templateName": "etmp-initial-consultation",
+        "currentScheduledAppointment": {
+            "eventType": "ENCOUNTER",
+            "eventTitle": "NEW ENC",
+            "startDateTime": start,
+            "endDateTime": end,
+            "hostId": "c08af975-8afc-49cb-84ac-7189b727148c",
+            "participants": [
+                {
+                    "participantId": "c08af975-8afc-49cb-84ac-7189b727148c",
+                    "participantRole": "psm",
+                },
+                {"participantId": patient.id, "participantRole": "patient"},
+            ],
+        },
+    }
+
+    encounter = patient.Encounter(**data)
+
+    created_enc = encounter.create()
```

Comment:
Just an FYI: You don't need to capture the output of `encounter.create()`. Calling `Resource.create()` makes the api call, updates `self`, then returns `self`. This is so you can use the method on an existing instance or create an instance and make an API call in one line:

```python
resource = welkin.Resource(id="foo")
resource.create()

# OR

resource = welkin.Resource(id="foo").create()
```

Reply:
ah good to know, thank you